src/main/markdown/meetingnotes.md

```diff
@@ -1,7 +1,63 @@
 Steering Committee Meeting Notes
 ================================
 
-## 07/20/2016
+## <a id="05/10/2017"></a> 05/10/2017
```

Comment:
Please use ISO 8601 (at least for the IDs); MDY is foreign/weird in most of the world.

https://en.wikipedia.org/wiki/Date_format_by_country